util/linuxfw/linuxfw.go

```diff
@@ -2,10 +2,254 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 // Package linuxfw returns the kind of firewall being used by the kernel.
+
+//go:build linux
+
 package linuxfw
 
-import "errors"
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/tailscale/netlink"
+	"tailscale.com/types/logger"
+)
+
+type ProtoVersion int
+
+const (
+	IPv4 ProtoVersion = iota
+	IPv6
+)
```

Comment:
Is this still used, it looks like it is used by methods that are unused, once we remove all of them, can these also go away?